doc/nrf/ug_nrf70_developing_powersave.rst

```diff
@@ -0,0 +1,323 @@
+Stations will be active only at TWT Service Period and remain in the sleep mode for the rest of the time.
+
+.. note::
+   The device is not expected to be active for the DTIM beacon that do not receive any buffered broadcast or multicast frames.
+   Applications are expected to keep note of this and set up TWT sessions based on their traffic profile.
+
+An access point has more control over the network in TWT mode and decides which device is going to transmit and when.
+The AP decides how many and which Resource Units (RU) are to be used (an RU is a contiguous set of subcarriers).
+TWT offers more efficient scheduling of transmissions.
+
+Devices can remain in the sleep mode even longer, if required.
+Before Wi-Fi 6, device would sleep for DTIM period and then awake, exchange data, and go back to the sleep mode for DTIM period and continues the same.
+With TWT, a device can sleep for seconds, minutes, or even hours.
+Some devices can be configured to communicate once a day to perform a transmission and sleep for the rest of the day.
+
+The following figure illustrates the TWT wakeup sequence:
+
+.. figure:: ../nrf/images/nRF70_ug_twt.svg
+   :alt: TWT wakeup sequence
+
+   TWT wakeup sequence
+
+Modes of operation in TWT mode
+==============================
+
+When the device is in the TWT mode, the following are the two modes of operation:
+
+* **Individual:** The client can choose when to wake up and sleep.
+  It can negotiate an agreement with the AP to wake up for receiving or transmitting the data.
+* **Broadcast:** The AP provides the schedule to all clients that support broadcast TWT.
+  This mode is not supported in the current release.
+
+Here is an example of a typical sequence of states in a TWT use case:
+
+1. Scan the network.
+#. Connect to an access point.
+#. Complete the application-level handshake.
+#. Set up a TWT session using the following command:
+
+   .. code-block:: console
+
+      wifi twt setup
+
+#. Tear down the TWT session.
+
+   * An application can tear down an ongoing TWT session and enter DTIM Power Save mode, if it is expecting group addressed frames and set up a TWT session again as applicable.
+   * Devices are not expected to schedule transmission outside the TWT Service Period.
+     An application can tear down an ongoing TWT session and schedule, if there is a requirement for immediate transmission.
+   * To tear down a TWT session, use the following commands:
+
+     .. code-block:: console
+
+        wifi twt teardown
+        wifi twt teardown_all
+
+Key parameters
+==============
+
+Following are the two key parameters of Target Wake Time:
+
+* **TWT Wake Duration**
+
+  * The minimum time that the TWT is requesting from the device.
+    The device needs to be active to complete the frame exchanges during TWT wake interval.
+  * Valid range for duration is 1 ms to 256 ms.
+  * The application must choose the right duration based on the traffic pattern.
+
+* **TWT Wake Interval**
```

Comment:
@udaynordic should we have an opening bullet of -
* Interval between successive TWT wake periods.

I think a diagram showing these would be more useful than the earlier one.

Reply:
@mjd90 , Yes. Mike. we can add a diagram showing TWT wake interval and duration.
@richabp , I will share the reference diagram.